sup3r/preprocessing/data_handling/dual_data_handling.py

```diff
@@ -113,7 +113,7 @@ def get_data(self):
         """Check hr and lr shapes and trim hr data if needed to match required
         relationship to lr shape based on enhancement factors. Then regrid lr
         data and split hr and lr data into training and validation sets."""
-        self._shape_check()
+        self._set_hr_data()
```

Comment:
Much better name

Reply:
this confused me so much hahaha

Reply:
Lol yeah it's a confusing method for sure

Reply:
Honestly just the name was confusing, I kept trying to figure out where hr_data was being set and took me a while to realize this method did that